requirements-common.txt

```diff
@@ -12,6 +12,7 @@ google-auth-oauthlib==1.2.1
 icalendar==5.0.13
 jellyfish==1.0.4                    # String Matching
 Jinja2==3.1.4                       # template engine for html files
+pandas==2.2.2
```

Comment:
I'm not sure, but I think we could use python list instead of pandas dataframe

Reply:
This allows to create a ready-to-use excel, added to the fact that it was a way I knew and it took me far less time than finding a new one